fulp_modules/features/antagonists/monster_hunter/ported_content/afterimage.dm

```diff
@@ -1,8 +1,230 @@
+////////////////////////
+//// PORTED HELPERS ////
```

Comment:
You know you don't need to be constantly mentioning everywhere in the code that stuff is ported, just makes it more annoying to read through. "Ported" ok but how is this relevant? And it doesn't even mention where it's from?